earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -77,6 +78,8 @@ void usage(const char *prog, const char *msg = 0, ...) {
      "    [--assetroot <dir>]       selected asset root or, modify the local\n"
      "                              path definition for an existing volume,\n"
      "                              or to add a volume definition.\n"
+     "  --listvolumes               List available volumes in the asset root.\n"
```

Comment:
Q: Should this be called, e.g., `--listhostvolumes` to differentiate listing the volumes on the current host from listing all defined volumes? Or, perhaps, this option can take a parameter indicating what host to list volumes for, with some parameter keywords for the current host and all hosts.

Or, perhaps, this can be left for the future, if the future default has the same behavior as this one.